MekHQ/src/mekhq/service/MassRepairOption.java

```diff
@@ -0,0 +1,182 @@
+        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "skillMin", getSkillMin());
+        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "skillMax", getSkillMax());
+        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "btnMin", getBthMin());
+        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "btnMax", getBthMax());
+        MekHqXmlUtil.writeSimpleXMLCloseIndentedLine(pw1, --indent, "massRepairOption");
+    }
+
+    public static List<MassRepairOption> parseListFromXML(Node wn2) {
+        List<MassRepairOption> massRepairOptions = new ArrayList<>();
+        NodeList mroList = wn2.getChildNodes();
+
+        for (int mroIdx = 0; mroIdx < mroList.getLength(); mroIdx++) {
+            Node mroNode = mroList.item(mroIdx);
+
+            if (mroNode.getNodeType() != Node.ELEMENT_NODE) {
+                continue;
+            }
+
+            try {
+                MassRepairOption mro = parseFromXML(mroNode);
+                if ((mro.getType() == -1) || (mro.getType() >= VALID_REPAIR_TYPES.length)) {
+                    MekHQ.getLogger().error("Attempted to load MassRepairOption with illegal type id of " + mro.getType());
+                } else {
+                    massRepairOptions.add(mro);
+                }
+            } catch (Exception e) {
+                MekHQ.getLogger().error("Failed to parse MassRepairOption from XML", e);
+            }
+        }
+
+        return massRepairOptions;
+    }
+
+    public static MassRepairOption parseFromXML(Node mroNode) {
+        MassRepairOption mro = new MassRepairOption(-1);
+
+        NodeList mroItemList = mroNode.getChildNodes();
+        for (int mroItemIdx = 0; mroItemIdx < mroItemList.getLength(); mroItemIdx++) {
+            Node mroItemNode = mroItemList.item(mroItemIdx);
+
+            if (mroItemNode.getNodeType() != Node.ELEMENT_NODE) {
+                continue;
+            }
+
+            if (mroItemNode.getNodeName().equalsIgnoreCase("type")) {
+                mro.setType(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("active")) {
+                mro.setActive(Integer.parseInt(mroItemNode.getTextContent().trim()) == 1);
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("skillMin")) {
+                mro.setSkillMin(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("skillMax")) {
+                mro.setSkillMax(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("btnMin")) {
+                mro.setBthMin(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            } else if (mroItemNode.getNodeName().equalsIgnoreCase("btnMax")) {
+                mro.setBthMax(Integer.parseInt(mroItemNode.getTextContent().trim()));
+            }
+
+            MekHQ.getLogger().info(String.format("massRepairOption %d.%s\n\t%s",
```

Comment:
Should this instead be `debug`?

Reply:
We currently use info for this and campaign options... but I'd say yes